__tests__/endpoints.test.js

```diff
@@ -96,6 +96,7 @@ describe('GET/api/articles/:article_id', ()=>{
                 votes: 100,
                 article_img_url:
                   "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                comment_count: '11'
```

Comment:
Please write a separate test for checking the comment count instead of doing it here in this test.